### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by maximilianschmitt** [d1ceward-on-dokku/plausible_on_dokku#19] [General]
Just wanted to say thanks for putting this repo together. It was super easy to follow the steps to get Plausible running on my Dokku instance. Well done!

**Comment by d1ceward** [comment]
Thanks, I'm happy to hear that it's useful to other people, but there are still improvements to be made in the guide, there have been a lot of changes recently with Dokku, I'll try to do that soon. 👍
